integrations/coinbase-integration/src/main/java/org/dash/wallet/integration/coinbase_integration/viewmodels/ConvertViewViewModel.kt

```diff
@@ -142,8 +142,13 @@ class ConvertViewViewModel @Inject constructor(
         }
         this._selectedCryptoCurrencyAccount.value = account
 
+        //To check if the user has different fiat than usd the min is 2 usd
+        val minFaitValue = "2".toBigDecimal() / account.currencyToUSDExchangeRate.toBigDecimal()
```

Comment:
What if the min fiat value changes?

Reply:
this is an assumption from our side I mean value is 2 USD

Reply:
we can address this in the next coinbase related story.